app/javascript/sdk/sdk.js

```diff
@@ -227,4 +227,8 @@ export const SDK_CSS = `
     width: 400px !important;
  }
 }
+
+.woot-widget-hidden {
```

Comment:
Let us name it bubble not widget. Until we come up with a better naming pattern, bubble is the button widget is the iFrameHolder

Reply:
Resolved